### PR TITLE
Progress Bar

### PR DESCRIPTION
**Pull request by gauriramesh**
-Added a progress bar that determines the confidence of your document by calculating the number of sentences that have unconfident words in them. The bar dynamically changes each time a word is accepted and goes to 100% when all changes are accepted. The bar also changes color depending on how far along the progress is.
-Tweaked CSS to make room for the progress bar. This included changing the positioning of some buttons and moving the instructions to the top of the page. (I also changed the instructions to be navy blue to go with our color scheme).

### CODE REVIEW

**Comment by ckrueger6** [comment]
As far as CSS changes go, I love the blue text and the progress bar looks good, but I think the buttons and icon need a bit more space from the border.

**Comment by ckrueger6** [comment]
I also am not sure that the purpose of the progress bar will be clear to or particularly helpful to the user.

**Comment by gauriramesh** [comment]
@ckrueger6 what about it is unclear? I put it in because I thought it would be nice to visually see a message get stronger when removing unconfident words.

**Comment by ckrueger6** [comment]
@gauriramesh There's no explanation that the bar shows how confident your message is. Also, there will be I'm sure a ton of cases where using the word "sorry" or "just" makes sense to the user and it might cause negative feelings if they're told their message in its final form is only 70% confident.

**Comment by allisoninman** [comment]
Maybe we should add in a way for users to ignore our suggestion and have it remove the warning class and increase the confidence progress bar so that they can still use the "under confident" words without the progress bar penalizing them? This is probably not the only solution and it is something we probably don't have time to do before moonshot but it's something to think about!

**Comment by gauriramesh** [comment]
@ckrueger6 Is there a way I can make it better that doesn't require just getting rid of the entire feature? I think it has the potential to serve a good purpose, which is why I put it in in the first place. I enjoyed making this feature and put a good amount of time into it, but I'd understand if we really want to get rid of it.

**Comment by ckrueger6** [comment]
I think @allisoninman has a good suggestion, and maybe a little explanation of what the bar means could go somewhere? I definitely don't want this feature to have wasted your time, I'm sorry I wasn't around this weekend to talk about the concept!

**Comment by gauriramesh** [comment]
Yeah, this was just a "nice to have" that I happened to have time to do, so I totally understand. We can talk about it at the meeting!

**Comment by gauriramesh** [comment]
@allisoninman I like the idea of being able to ignore a suggestion. Even outside the context of a progress bar, users might get apprehensive about clicking the "copy to clipboard" button if there's still formatting on the text/if they have to manually change it themselves

File: analyzeText.js [hunk 1]
```diff
@@ -99,4 +106,74 @@ getKeywordMatches = (keywords)=>{
...
+    }
+    return initialAnalysis;
+}
+
+//Currently calculates percentages based on number of sentences that contain a keyword.
+getPercentages = () => {
+    let input  = $("#popupMessage").html().toString();
+    input = removeCode(input);
+    let percent = 0;
+    let sentenceArr = input.match(/\S.*?\."?(?=\s|$)/g);
```
Comment (ckrueger6) [inline analyzeText.js]: Does this account for "!" as a sentence ending?

**Comment by gauriramesh** [comment]
@allisoninman @ckrueger6 what do y'all think about an "Ignore Remaining Suggestions" button that would take the remaining unaccepted suggestions, remove the formatting, and increment the progress bar to 100%?

**Pull request closed without merging**